sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/FormRecognizerRecordedTestSanitizer.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core.Testing;
+
+namespace Azure.AI.FormRecognizer.Tests
+{
+    public class FormRecognizerRecordedTestSanitizer : RecordedTestSanitizer
+    {
+        public override void SanitizeHeaders(IDictionary<string, string[]> headers)
+        {
+            if (headers.ContainsKey(Constants.AuthorizationHeader))
+            {
+                headers[Constants.AuthorizationHeader] = new[] { SanitizeValue };
+            }
+        }
+
+        public override string SanitizeVariable(string variableName, string environmentVariableValue)
+        {
+            return variableName switch
+            {
+                "FORM_RECOGNIZER_API_KEY" => SanitizeValue,
+                "FORM_RECOGNIZER_BLOB_CONTAINER_SAS_URL" => SanitizeSasToken(environmentVariableValue),
```

Comment:
Do we need to preserve any part of this value to be able to playback?

Reply:
Not really, you have a point. The SAS URI is part of some request bodies, though. We can completely replace it as long as we apply the same changes to the ones in request bodies once they're enabled.